src/handlers/mod.rs

```diff
@@ -115,7 +116,7 @@ impl <D: CallbackDataWithPrefix> HandlerImplResult<D> {
 
 pub fn reply_html<T: Into<String>>(bot: Bot, msg: &Message, answer: T) -> JsonRequest<SendMessage> {
     // TODO: split to several messages if the answer is too long
-    let mut answer = bot.send_message(msg.chat.id, answer);
+    let mut answer = bot.send_message(msg.chat.id, answer).disable_link_preview(true);
```

Comment:
Let's use the same style for both expressions:
```rust
let mut answer = bot.send_message(msg.chat.id, answer)
    .parse_mode(Html)
    .disable_link_preview(true);
```

Reply:
done 👍